vector/src/main/AndroidManifest.xml

```diff
@@ -306,7 +306,9 @@
             android:supportsPictureInPicture="true" />
 
         <activity android:name=".features.terms.ReviewTermsActivity" />
-        <activity android:name=".features.widgets.WidgetActivity" />
+        <activity android:name=".features.widgets.WidgetActivity"
```

Comment:
ideally we'd let the system reconfigure but avoid calling `loadUrl` on the webview on each creation _buttt_ as we're already using this pattern +1: